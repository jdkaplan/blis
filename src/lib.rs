pub mod bytecode;
pub mod compile;
pub mod parse;
pub mod runtime;
pub mod vm;

#[cfg(test)]
mod testing;
