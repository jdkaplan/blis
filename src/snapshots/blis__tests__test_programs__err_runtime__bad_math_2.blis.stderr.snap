---
source: src/lib.rs
description: "println(1 + 2.0);\n"
input_file: test_programs/err_runtime/bad_math_2.blis
---

